lib/Charts/LineChart.js

```diff
@@ -260,18 +272,18 @@ function render(element, state) {
     }
 
     g.select('.x.axis')
-        .transition('1st').duration(transitionDuration)
+        .transition(t)
```

Comment:
What does it mean to use the same transition `t` here as elsewhere, especially since this one had a specific name before?